stl/src/vector_algorithms.cpp

```diff
@@ -884,7 +884,8 @@ namespace {
                             _BitScanForward(&_H_pos, _Mask); // lgtm [cpp/conditionallyuninitializedvariable]
 
                             const auto _V_pos = _Traits::_Get_v_pos(_Cur_idx_min, _H_pos); // Extract its vertical index
-                            _Res._Min         = _Base + _V_pos * 16 + _H_pos; // Finally, compute the pointer
+                            _Res._Min =
+                                _Base + static_cast<size_t>(_V_pos) * 16 + _H_pos; // Finally, compute the pointer

```

Comment:
https://github.com/microsoft/STL/pull/3619#discussion_r1156912405

> Sorry, I forgot about x86, where this can truncate from `unsigned long long` to `size_t`. Have to go back to `static_cast`.